src/ports/postgres/modules/pmml/test/pmml_logistic.sql_in

```diff
@@ -0,0 +1,138 @@
+\i m4_regexp(MADLIB_LIBRARY_PATH, `\(.*\)/lib',
```

Comment:
I deleted `src/ports/postgres/modules/pmml/test/table_to_pmml.sql_in` and created two files out of it: `logistic.sql_in` and `linear.sql_in`